includes/Settings.php

```diff
@@ -438,6 +438,11 @@ public function register(): void {
 	 * @return string|array<int|string,mixed>|bool|int Setting value.
 	 */
 	public function get_setting( string $key, $default_value = false ) {
+		if ( \function_exists( 'wp_prime_option_caches_by_group' ) ) {
+			wp_prime_option_caches_by_group( self::SETTING_GROUP );
+			wp_prime_option_caches_by_group( self::SETTING_GROUP_EXPERIMENTS );
+		}
```

Comment:
A bit wasteful to call `wp_prime_option_caches_by_group` every time `get_setting()` is used.

Can we perhaps just call it once on `init` or something?